themes/notmyidea/static/css/main.css

```diff
@@ -183,7 +183,7 @@ img.left, figure.left {float: left; margin: 0 2em 2em 0;}
 	/* Main Nav */
 	#banner nav {
 		background: #000305;
-		font-size: 1.143em;
+		font-size: 1.09em;
```

Comment:
I'm not sure we'd want to reduce our font size; we can definitely widen the title bar. See #10 for how we did that last time. It would be good to have this change in a separate commit.